src/test/java/com/google/api/codegen/testdata/nodejs_main_library.baseline

```diff
@@ -292,6 +292,18 @@ LibraryServiceApi.matchBookFromArchivedBookName =
  * @returns {gax.EventEmitter} - the event emitter to handle the call
  *   status.
  * @throws an error if the RPC is aborted.
+ *
+ * @example
+ *
+ * var api = libraryV1.libraryServiceApi;
```

Comment:
That's new calling style based on the discussion between @jmuk and gcloud team. This is also something special from NodeJS.